packages/swap/src/handlers/getSwapStatus.ts

```diff
@@ -0,0 +1,191 @@
+import assert from 'assert';
```

Comment:
for the most part this a copy of `/routes/swap.ts`

Reply:
can we reuse this in `/routes/swap.ts` like we did with the `openSwapDepositChannel`

Reply:
we could but its not so simple because we'd need to serialize bigints. prefer to just switch over when we do